credential-info-modal-directive.js

```diff
@@ -0,0 +1,32 @@
+/*!
```

Comment:
Yes, I figured the code would be easier to deal with if all modals here used the same style. They could all be converted to new style at once later. It was also of course easier to just copy and edit an existing modal.